packages/dds/tree/src/test/shared-tree/opSize.bench.ts

```diff
@@ -496,8 +468,17 @@ describe("Op Size", () => {
 		for (const { description, style, extraDescription } of styles) {
 			describe(description, () => {
 				for (const { percentile, word } of sizes) {
-					it(`${BENCHMARK_NODE_COUNT} ${word} nodes in ${extraDescription}`, () => {
-						benchmarkOps(style, percentile);
+					benchmarkCustom({
+						only: false,
+						type: BenchmarkType.Measurement,
+						title: `${BENCHMARK_NODE_COUNT} ${word} nodes in ${extraDescription}`,
+						run: async (reporter) => {
+							benchmarkOps(style, percentile);
+							const opStats = getOperationsStats(currentTestOps);
+							for (const key of Object.keys(opStats)) {
+								reporter.addMeasurement(key, opStats[key]);
+							}
+						},
 					});
```

Comment:
Since you have this set of code called repeatedly in the same way I would extract it into an extra helper function.

Reply:
Response left in another PR. 

https://github.com/microsoft/FluidFramework/pull/22140